pkg/controller/atlasproject/cloud_provider_integration.go

```diff
@@ -67,12 +69,14 @@ func ensureCloudProviderIntegration(workflowCtx *workflow.Context, project *mdbv
 }
 
 func syncCloudProviderIntegration(workflowCtx *workflow.Context, projectID string, cpaSpecs []mdbv1.CloudProviderIntegration) (bool, error) {
-	atlasCPAs, _, err := workflowCtx.Client.CloudProviderAccess.ListRoles(workflowCtx.Context, projectID)
+	atlasCPAs, _, err := workflowCtx.SdkClient.CloudProviderAccessApi.
+		ListCloudProviderAccessRoles(workflowCtx.Context, projectID).
+		Execute()
 	if err != nil {
 		return false, fmt.Errorf("unable to fetch cloud provider access from Atlas: %w", err)
 	}
 
-	AWSRoles := sortAtlasCPAsByRoleID(atlasCPAs.AWSIAMRoles)
+	AWSRoles := sortAtlasCPAsByRoleID(atlasCPAs.GetAwsIamRoles())
```

Comment:
[Go convention on initialisms](https://go.dev/wiki/CodeReviewComments#initialisms) is for this to be called:

```suggestion
	AWSRoles := sortAtlasCPAsByRoleID(atlasCPAs.GetAWSIAMRoles())
```

Reply:
@josvazg, I know and usually follow it, but Go Atlas SDK decided to use a different convention and your suggestion is on a method from them. =/

Reply:
Nothing we can do then. Sorry for the noise!